modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Statement.scala

```diff
@@ -69,7 +69,7 @@ object Statement {
   case class GetVersion(tableName: String) extends Statement
   case class RenameTable(from: String, to: String) extends Statement
 
-  case object SetSchema extends Statement
+  case object SetSearchPath extends Statement
```

Comment:
Yes it's still needed!  I was hoping I could get rid of it, but I can't.

It's because Redshift requires us to set the search path before calling [this line here](https://github.com/snowplow/snowplow-rdb-loader/blob/4.0.4/modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/Redshift.scala#L197).

Annoyingly, we don't need it for Snowflake or Databricks.  I would prefer to remove it for Snowflake and Databricks, but I couldn't find a neat way to do it for just one target.